guice-intro/src/main/java/com/baeldung/examples/guice/marker/Communicator.java

```diff
@@ -0,0 +1,9 @@
+package com.baeldung.examples.guice.marker;
+
+/**
+ *
+ * @author Baeldung
+ */
+public interface Communicator {
+    

```

Comment:
This was meant to be a marker interface, just to illustrate the point of selective injection